imod/tests/test_mf6/test_multimodel/test_mf6_partitioning_unstructured.py

```diff
@@ -405,3 +405,23 @@ def test_partitioning_unstructured_with_well(
     # np.testing.assert_allclose(
     #     cbc["chd"].values, original_cbc["chd"].values, rtol=1e-5, atol=1e-3
     # )
+
+
+@pytest.mark.usefixtures("circle_model_transport")
+@parametrize_with_cases("partition_array", cases=PartitionArrayCases)
+def test_partition_transport(    tmp_path: Path,
+    circle_model_transport: Modflow6Simulation,partition_array: xu.UgridDataArray):
```

Comment:
Formatting looks off here, did ruff produce this?

Reply:
yes